arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -174,8 +181,8 @@ def start_pcp(
             pcp2csv_cmd.extend(metrics)
 
         max_retries = 1
-        pcp2csv_return = ("", "")
-        pcp2json_return = ("", "")
+        pcp2csv_status = ("", "")
+        pcp2json_status = ("", "")
```

Comment:
This looks odd. Below we have `pcp2csv_status, pcp2csv_return = run_oneshot_cmd(pcp2csv_cmd)`, which sets each value to a string from the tuple returned by the function. Yet here we're initializing each `*_status` to a tuple.

Apparently this doesn't really matter since your logic expects that the calls will be made, fixing the types and defining the `*_return` variants ... but this still looks odd.

Reply:
I don't like this either, and I'm sure there is a better approach, but the reason for the initialization is that there isn't a guarantee in the logic below that either `*_status` value will be set since they have some mutually-exclusive conditional states.

Reply:
But one or the other or both _should_ be set by the logic.

Reply:
I don't think an initial definition for the `*_return` values is important for the logic, but, yeah, Webb's suggestion would work.

My point is that the `*_status` values should never be *tuples* ... they should be strings with `"success"` or `"error"`.